packages/api-server/src/methods/gw-error.ts

```diff
@@ -47,6 +48,11 @@ export function parseGwError(error: any): GwErrorDetail {
   if (message.startsWith(prefix)) {
     const jsonErr = message.slice(prefix.length);
     const err = JSON.parse(jsonErr);
+
+    if (err.data == null) {
+      parseGwRpcError(error);
```

Comment:
Why parse gw rpc error here ?

Reply:
the original code here assume `err.data` is not null and try getting gas_used value from the error. but err.data sometimes will be null, in that way we can not get gas_used value from error, so should parse error and return clear error info to user.